### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by patryk-babon**
- [DEMO LINK](https://patryk-babon.github.io/layout_catalog/)
- [TEST REPORT LINK](https://patryk-babon.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/utils/_variables.scss
Comment (choeqq) [inline src/styles/utils/_variables.scss:1]: please consider using `font-size` variables

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: $card-width;
+  padding: 32px 16px 16px;
+  background-color: $color-white-accent;
+  border: 1px solid $color-white-anti-flash;
+  border-radius: 5px;
+  transition: 0.3s;
```
Comment (choeqq) [inline src/styles/blocks/card.scss:10]: please list properties you wanna use transition on

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin-top: 16px;
+    color: $color-white-accent;
+    font-size: 14px;
```
Comment (choeqq) [inline src/styles/blocks/card.scss:70]: consider using `font-size` variable

**Review by choeqq — CHANGES_REQUESTED** [review]
Looking really good! Little adjustments are needed 🚀

**Comment by patryk-babon** [comment]
> Looking really good! Little adjustments are needed 🚀

Done!

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__title {
+      color: #34568b;
+    }
+  }
```
Comment (JoyCoffeeAddict) [inline src/styles/blocks/card.scss:93]: Put `:hover` styles related to `.card` immediately after it.

**Review by JoyCoffeeAddict — APPROVED** [review]
One small mistake, but I trust that you will fix it.

File: src/styles/blocks/card.scss (hunk 3 above)
Comment (patryk-babon) [reply src/styles/blocks/card.scss]: Fixed! 👍

**Pull request closed without merging**